### ISSUE TITLE
public_folder forces leading /

### ISSUE THREAD

**Issue by Larsklopstra** [netlify/netlify-cms#2238]
**Describe the bug**
Using `public_folder: "../uploads"` I'd assume it would return `../uploads/image.png` but the result is `./../uploads/image.png` thus resulting in a 404. 

The forced leading slash will prevent [Gridsome](https://gridsome.org) from optimizing the images.

**Expected behavior**
Using `public_folder: "../uploads"` should return `../uploads/image.png` **NOT** `./../uploads/image.png`.

**CMS configuration**
```
backend:
  name: github
  repo: Larsklopstra/gridsome-starter-netlify-cms

media_folder: "uploads"
public_folder: "../uploads"

collections:
  - name: "posts"
    label: "Posts"
    folder: "posts"
    create: true
    slug: "{{slug}}"
    fields:
      - {label: "Title", name: "title", widget: "string"}
      - {label: "Excerpt", name: "excerpt", widget: "string"}
      - {label: "Body", name: "body", widget: "markdown"}
```

**Additional context**
Using Gridsome as static site generator.

**Comment by tomrutgers** [comment]
I'm not exactly a gridsome expert, but the paths in your config need to be absolute. Pretty sure you don't want them to be in root either. So something like:
```
media_folder: "static/uploads/"
public_folder: "/uploads/
```

**Comment by Larsklopstra** [comment]
@tomrutgers I want to avoid placing files into the static folder, if I change the `article-example.md` its contents from `![](/../image.png)` to `![](../image.png)` then it will work perfectly, so the issue is the forced leading slash. I've tried many different configs.

**Comment by erquhart** [comment]
Related to #325 and #1668.

**Comment by erquhart** [comment]
Resolved by #2958.

Docs here: https://www.netlifycms.org/docs/beta-features/#folder-collections-media-folder
